dandiapi/api/views/asset.py

```diff
@@ -135,6 +135,29 @@ def _paginate_asset_paths(
     return {'results': paths.data, 'count': asset_count, 'next': next_url, 'previous': prev_url}
 
 
+def _maybe_validate_asset_metadata(asset: Asset):
```

Comment:
what is conditional about this method?

Reply:
If this is a zarr asset, it doesn't validate metadata